### PR TITLE
chore(loaders): update prop descriptions

### PR DESCRIPTION
**Pull request by Mikaelia**
## Description

Updates loaders package prop descriptions

### CODE REVIEW

File: packages/loaders/src/elements/Dots.tsx [hunk 1]
```diff
@@ -21,21 +21,13 @@ import {
...
   size?: string | number;
-  /**
-   * Sets the fill color. Inherits the parent font `color` by default
-   **/
+  /** Sets the fill color. Inherits the parent's `color` by default. */
   color?: string;
-  /**
-   * Sets the speed of the loading animation in milliseconds
-   **/
+  /** Sets the length of the animation cycle in milliseconds **/
```
Comment (jzempel) [inline packages/loaders/src/elements/Dots.tsx:28]: omg, this is so much better. Solves a _long_ confusion.

File: packages/loaders/src/elements/Progress.tsx [hunk 2]
```diff
@@ -14,9 +14,12 @@ const COMPONENT_ID = 'loaders.progress';
 export interface IProgressProps extends HTMLAttributes<HTMLDivElement> {
   /** Sets the progress as a value between 0 and 100 */
   value?: number;
-  /** Sets the foreground bar fill color. Defaults to the `successHue` theme value */
+  /**
+   * Sets the foreground bar's fill color.
+   * Defaults to the `successHue` [theme](https://garden.zendesk.com/components/theme-object#default_theme) value.
```
Comment (jzempel) [inline packages/loaders/src/elements/Progress.tsx:19]: Let's go with `/components/theme-object#colors` here.

File: packages/loaders/src/elements/Spinner.tsx [hunk 3]
```diff
@@ -47,19 +47,19 @@ const computeFrames = (
...
    **/
   duration?: number;
   /**
-   * Sets the fill color. Inherits the parent font `color` by default
+   * Sets the fill color. Inherits the parent's `color` by default.
    **/
   color?: string;
   /**
-   * Delays showing the loader to prevent a quick render flash during normal loading times. Time is in milliseconds
+   * Delays displaying the loader to prevent a quick render flash during normal loading times
```
Comment (jzempel) [inline packages/loaders/src/elements/Spinner.tsx:62]: Should ~quick~ be removed per your previous description?

**Review by jzempel — CHANGES_REQUESTED** [review]

File: packages/loaders/src/elements/Dots.tsx (hunk 1 above)
Comment (Mikaelia) [reply packages/loaders/src/elements/Dots.tsx:28]: 🎉

File: packages/loaders/src/elements/Spinner.tsx (hunk 3 above)
Comment (Mikaelia) [reply packages/loaders/src/elements/Spinner.tsx:62]: Agreed

File: packages/loaders/src/elements/Progress.tsx [hunk 4]
```diff
@@ -14,9 +14,12 @@ const COMPONENT_ID = 'loaders.progress';
 export interface IProgressProps extends HTMLAttributes<HTMLDivElement> {
   /** Sets the progress as a value between 0 and 100 */
   value?: number;
-  /** Sets the foreground bar fill color. Defaults to the `successHue` theme value */
+  /**
+   * Sets the foreground bar's fill color.
+   * Defaults to the `successHue` [theme](/components/theme-object#default_theme) value.
```
Comment (jzempel) [inline packages/loaders/src/elements/Progress.tsx:19]: This URL would be better as `/components/theme-object#colors`

**Review by jzempel — CHANGES_REQUESTED** [review]

**Review by jzempel — APPROVED** [review]

**Pull request merged**